docs/analysis/va.mdx

```diff
@@ -0,0 +1,125 @@
+---
```

Comment:
why do we have 2 variance anlaysis files?

Reply:
Deleted the absolute file "va.mdx" after confirming with the QA team.